cli.js

```diff
@@ -2,10 +2,15 @@
 'use strict';
 const meow = require('meow');
 const opn = require('opn');
+const getStdin = require('get-stdin');
+const tempWrite = require('temp-write');
+const dataUriToBuffer = require('data-uri-to-buffer');
```

Comment:
Premature optimization is the root of all evil.

http://c2.com/cgi/wiki?PrematureOptimization

A few requires have no noticeable impact.